ci/initializeWallet.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+echo 'Downloading latest witnet node release...'
+
+wget $(curl -s https://api.github.com/repos/witnet/witnet-rust/releases/latest | grep 'browser_' | cut -d\" -f4)
+
+tar -zxvf witnet-0.6.0-x86_64-unknown-linux-gnu.tar.gz
```

Comment:
This will break on witnet `0.6.1`
```suggestion
tar -zxvf witnet-*-x86_64-unknown-linux-gnu.tar.gz
```